app/login/styles/index.scss

```diff
@@ -19,17 +19,9 @@
     left: 0;
     margin-left: 0;
   }
-  .login__logo {
```

Comment:
Does this still look okay with the IBM logo?

Reply:
### **Original login screen**
![image](https://user-images.githubusercontent.com/63627630/125717247-66dcf6e1-8e62-46de-9c1e-a4fc48ddffdd.png)



### **Use this modified login screen**
1. **No** keyword in VPD 
![image](https://user-images.githubusercontent.com/63627630/125717921-fa2d9ccb-4ff6-4ba4-b94d-57c5f9a20f42.png)


2. Keyword in VPD
![image](https://user-images.githubusercontent.com/63627630/125718038-f19e7b0a-42c2-4567-a4a6-1be033e8ab70.png)